lisp/ess-julia.el

```diff
@@ -39,224 +39,14 @@
 
 (require 'compile); for compilation-* below
 (require 'ess-utils)
+(require 'julia-mode)
```

Comment:
@vspinu @mmaechler 

The code for generating tests stopped working because it cannot load `julia-mode`. I installed the package but it still cannot find it.

Would it be reasonable to wrap this as follows?
```lisp
(unless (bound-and-true-p 'ess-test-script)
  (require 'julia-mode))
```
Or is it a problem with my Emacs installation that I should solve?